ContactManager2/src/components/Contact.jsx

```diff
@@ -0,0 +1,20 @@
+import PropTypes from "prop-types"
+import React from "react"
+
+export const Contact = React.memo(function Contact({ contact, deleteContact }) {
```

Comment:
no need to use React.memo here

Reply:
This is to prevent re-rendering of all the items in the list when a new item is added.